include/open62541/server.h

```diff
@@ -1886,6 +1891,22 @@ typedef struct {
 UA_ServerStatistics UA_EXPORT
 UA_Server_getStatistics(UA_Server *server);
 
+/* Reverse connect */
+typedef enum {
+    REVERSE_CONNECT_STATUS_INITIAL = 0,
+    REVERSE_CONNECT_STATUS_CONNECTING,
+    REVERSE_CONNECT_STATUS_CONNECTED,
```

Comment:
This refers to the secure channel.
Each retry will start with a callback for CONNECTING followed by a callback for CONNECTED or DISCONNECTED.

I'd personally prefer to keep reconnecting as long as the reverse connect is not removed by the user.
Otherwise users would need to keep book about which handle is for which URL even if they just care about the client being able to communicate with the server.

Reply:
So there is no real difference between a fresh and a disconnected reverse connect state.